erpnext/shopping_cart/cart.py

```diff
@@ -86,6 +85,29 @@ def place_order():
 				if item.qty > item_stock.stock_qty[0][0]:
 					throw(_("Only {0} in Stock for item {1}").format(item_stock.stock_qty[0][0], item.item_code))
 
+	#if checkout without payment has been enabled, submit the quotation, and convert to sales order
+	if cart_settings.sales_team_order_without_payment:
+	
+		quotation.requested_delivery_date = getdate(delivery_date)
```

Comment:
Have you added requested_delivery_date to Quotation doctype? I don't see any JSON changes on Quotation in this PR

Reply:
I did do it but I have no clue why its not showing on the json, its working fine on the backend though. I'll look into it